lib/scrub_fields.rb

```diff
@@ -229,21 +224,26 @@ def govdoc(str)
 
   # DRY code for the status, condition and govdoc functions
   def simple_matcher(rx, str)
-    output = []
-    str.strip!
-    m = rx.match(str)
-    output << m[0] unless m.nil?
-
     # Get the name of the calling method
-    cmeth = caller_locations[0].label
+    cmeth  = caller_locations[0].label
```

Comment:
Consider moving cmeth to a parameter with a default value, e.g.

```ruby
def simple_matcher(rx, str, label=caller_locations[0].label
```

Which is to say, it seems like the important part is that we'd like some way to refer to what it is we're matching, not that it happens to be the calling method -- but it's fine to have it default to the name of the calling method.

I also wonder a bit if it might make sense to extract these to (tiny) classes `StatusMatcher`, `ConditionMatcher`, ... - that would also give a better way to hold the constants than as constants in `ScrubFields`. Not worth doing now, but maybe worth doing rather than adding additional regexes to `ScrubFields`/